README.md

```diff
@@ -142,6 +142,7 @@ ng-s3upload allows to customize the directive template using themes. Currently t
 ```javascript
 app.config(function(ngS3Config) {
   ngS3Config.theme = 'bootstrap3';
+  ngS3Config.theme = 'path/to/custom/theme.html';
```

Comment:
Can you please update the description accordingly, explain how to use this feature.